### DISCUSSION TITLE
Notion database integration

### DISCUSSION THREAD

**Discussion by PedroDSFerreira** [bergercookie/syncall#100] [Ideas]
Are there any plans to implement sync integrations with a notion database? Having the tasks in a db allows for much more versatility, compared to having checkboxes in a page.

**Comment by bergercookie** [comment]
It's definitely on my mind but I haven't had time to work on it.
Feel free to contribute a prototype if interested :)

**Comment by bergercookie** [comment]
Just added a ticket re  this, https://github.com/bergercookie/syncall/issues/105.
